scripts/migrate-token-pool.js

```diff
@@ -0,0 +1,42 @@
+import pg from 'pg'
+import RedisTokenPersistence from '../core/token-pooling/redis-token-persistence.js'
+
+let redisUrl, postgresUrl
+
+try {
+  redisUrl = process.argv[2]
+  postgresUrl = process.argv[3]
```

Comment:
For this script it is useful to pass the connection strings in on the console instead of using the application config because it allows us to copy the data over before we set `POSTGRES_URL` to switch the pool over.

Also I'm not too worried about achieving perfection in this specific script. It is a file we are going to use once and then delete it.